qiskit/transpiler/preset_passmanagers/level3.py

```diff
@@ -52,20 +43,9 @@
 from qiskit.transpiler.passes import Collect2qBlocks
 from qiskit.transpiler.passes import ConsolidateBlocks
 from qiskit.transpiler.passes import UnitarySynthesis
-from qiskit.transpiler.passes import ApplyLayout
-from qiskit.transpiler.passes import CheckGateDirection
-from qiskit.transpiler.passes import TimeUnitConversion
-from qiskit.transpiler.passes import ALAPScheduleAnalysis
-from qiskit.transpiler.passes import ASAPScheduleAnalysis
-from qiskit.transpiler.passes import ConstrainedReschedule
-from qiskit.transpiler.passes import InstructionDurationCheck
-from qiskit.transpiler.passes import ValidatePulseGates
-from qiskit.transpiler.passes import PulseGates
-from qiskit.transpiler.passes import PadDelay
 from qiskit.transpiler.passes import Error
-from qiskit.transpiler.passes import ContainsInstruction
+from qiskit.transpiler.preset_passmanagers import common
 from qiskit.transpiler.passes.layout.vf2_layout import VF2LayoutStopReason
-from qiskit.transpiler.passes.layout.vf2_post_layout import VF2PostLayoutStopReason
 
 from qiskit.transpiler import TranspilerError
 
```

Comment:
In line 53, 
```
level_3_pass_manager(pass_manager_config: PassManagerConfig) -> StagedPassManager:
```

Reply:
The note in line 67:
```
    Note:
        In simulators where ``coupling_map=None``, only the unrolling and
        optimization stages are done.
```

It might be confusing now, since what is called `stage` is a very concrete thing.  The same applies to the other levels.

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/6403/commits/a2472f29b01d19a8ee55118d5533e507d6cda871 and I removed that comment from the docs in https://github.com/Qiskit/qiskit-terra/pull/6403/commits/c5ac5f69f2006cbb43afe184ac63027f9394613b I thought about updating it, but it just felt better to remove it. Especially since ``coupling_map`` wasn't an arg anymore and also there are backends where it's not explicitly set anymore either.